pytensor/tensor/rewriting/linalg.py

```diff
@@ -242,26 +223,25 @@ def no_transpose_symmetric(fgraph, node):
 
 
 @register_stabilize
-@node_rewriter([Blockwise])
+@node_rewriter([blockwise_of(OpPattern(Solve, b_ndim=2))])
 def psd_solve_with_chol(fgraph, node):
     """
     This utilizes a boolean `psd` tag on matrices.
     """
-    if isinstance(node.op.core_op, Solve) and node.op.core_op.b_ndim == 2:
-        A, b = node.inputs  # result is solution Ax=b
-        if getattr(A.tag, "psd", None) is True:
-            L = cholesky(A)
-            # N.B. this can be further reduced to a yet-unwritten cho_solve Op
-            #     __if__ no other Op makes use of the L matrix during the
-            #     stabilization
-            Li_b = solve_triangular(L, b, lower=True, b_ndim=2)
-            x = solve_triangular((L.mT), Li_b, lower=False, b_ndim=2)
-            return [x]
+    A, b = node.inputs  # result is solution Ax=b
+    if getattr(A.tag, "psd", None) is True:
+        L = cholesky(A)
+        # N.B. this can be further reduced to a yet-unwritten cho_solve Op
+        #     __if__ no other Op makes use of the L matrix during the
+        #     stabilization
```

Comment:
I know you only changed the indentation level and thus don't care, but this Op exists now, so this note is out of date.

Reply:
I think so. I'll make an issue somewhere else